src/mixins/serverData.js

```diff
@@ -20,6 +20,22 @@
  *
  */
 
+/*
+ * This file provides global methods for using the serverData structure
+ * the serverData structure's data are initialy found in a DOM element
+ *  provided by the server.
+ *
+ * It may contain the following information:
```

Comment:
How about a jsdoc type definition for this? Then your IDE could give you hints when coding.

Reply:
ok. I'll do that